compositor_render/src/transformations.rs

```diff
@@ -1,3 +1,4 @@
+pub mod common;
```

Comment:
if you change name in api, chaneg the code too to use that name. e.g.
```suggestion
pub mod builtin;
```
or
```suggestion
pub mod builtin_transforms;
```